app/common/elasticsearchclient.py

```diff
@@ -240,19 +240,25 @@ def best_hit_search(self,
         # current_app.logger.debug("Got %d Hits in %ims" % (res['hits']['total'], res['took']))
         data = []
         
-        for res in results['responses']:
-            res_data = [];
-            for hit in res['hits']['hits']:
```

Comment:
Yes, I agree - in my explanation about fuzzy match I should have specified that it is for the cases when the name provided in file is not the actual ENS-Id, but really the name. When ENS id is provided - login is different and I did use that logic in the webapp.